### PR TITLE
Terminal Formatting

### PR DESCRIPTION
**Pull request by MaxWeisen**
Terminal formatting has been finalized as well as the font size. We ran Jest commands to confirm testing functionality works. We are ready for custom commands to be sent to the terminal.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +56,7 @@
     "build": "NODE_ENV=production react-app-rewired build && npm run electron-build",
     "react-build": "NODE_ENV=production react-scripts build",
     "electron-build": "NODE_ENV=production electron-builder -mwl",
-    "electron-rebuild": "electron-rebuild -f -w node-pty",
+    "electron-rebuild": "electron-rebuild",
```
Comment (dieunity) [inline package.json:59]: Let's keep electron-rebuild flags

**Review by dieunity — CHANGES_REQUESTED** [review]
let's keep the electron rebuild flags for node pty on line 60 of the old code- Dieu

**Review by dieunity — APPROVED** [review]
awesome!

**Review by JIB3377 — APPROVED** [review]
Great!

**Pull request merged**